system/View/Plugins.php

```diff
@@ -90,6 +98,8 @@ public static function ValidationErrors(array $params = []): string
     /**
      * Wrap helper function to use as view plugin.
      *
+     * @param list<int|string> $params
```

Comment:
There seems no way to set int values from templates.
```suggestion
     * @param list<string> $params
```